nodejs_conductor/publish.js

```diff
@@ -58,7 +64,7 @@ npmPackageJson.scripts.fallback = "npm run compile && mkdir -p bin-package && cp
 
 fs.writeFileSync("./dist/package.json", JSON.stringify(npmPackageJson, null, 2));
 
-shell.mkdir("./bin-package");
+shell.mkdir('-p', "./bin-package");
```

Comment:
what's -p?

Reply:
recursive -- the easiest way to make it not complain if the dir already exists, but still create it if it doesn't